### PR TITLE
display cohort by entry and grad date

### PR DESCRIPTION
**Pull request by VKong6019**
Issue: https://github.com/sandboxnu/pharmd-tracker/issues/41

Per the issue description, I went with formatting the `cohort` on the frontend and repurposed the `CohortField` component that didn't seem to be used anywhere except in stories. The changes should be reflected in both the student lists and quick view panel, but do let me know if there are other parts that should also reflect this change.

The component will take the last two characters of entry and grad date to format into cohort, assuming both dates end in years (e.x. 2018 and 2023).

### CODE REVIEW

**Comment by sophiacrennan** [comment]
Looks good! Can you also reflect this change in the cohort filter. Right now if I filter by a specific filter, no results show up.

**Comment by VKong6019** [comment]
currently, the cohort filter would work on a single cohort, and needs some backend fix to handle multiple cohort filters

that should be a quick fix, so i can also do that as well

File: pharmd-app/src/components/Fields/CohortField.js [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
-    <Field>
-      {cohort.current}
-      {cohort.moved && (
-        <Offset>
-          <InfoIcon src={ErrorIcon} text={`Original: ${cohort.original}`} />
-        </Offset>
-      )}
-    </Field>
-  );
+  return <Field>{cohort}</Field>;
```
Comment (emaela) [inline pharmd-app/src/components/Fields/CohortField.js]: Would it be possible/beneficial to use a field component from react-admin here? I think the FunctionField from react-admin might be applicable here, what do you think?

Comment (VKong6019) [reply pharmd-app/src/components/Fields/CohortField.js]: Im open to using FunctionField, I'm not too sure what the difference is. So I'll look into it more (we love documentation)

**Review by sophiacrennan — APPROVED** [review]
Looks great! Thanks :)

**Pull request merged**